### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Loltolo-Lesapiti**
In this project, the following additions are made to the portfolio mobile version.

- Added contact section.
- Added a form to the contact sections.
- And finally added name, email, and the text area fields to the form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        </ul>
+    </section>
+    <!--Footer section-->
+    <footer class="contact_section">
+        <div class="footer_content">
+            <h2 class="contact-header">Contact me</h2>
+            <p class="contact-message">
+                If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+            </p>
+            <form class="form-container" action="#" method="POST">
```
Comment (aboongm) [inline index.html]: - `action` value should be similar to `https://formspree.io/f/{form_id}`. You have to sign up to formspree.io and use it in the form. And when you click on the submit button, the form details should be sent to the formspree website.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                <input id="name" type="text" maxlength="30" name="full-name" placeholder="Petro Lesapiti" required class="input">
+                <input id="email" type="email" name="email-address" placeholder="petrolesapiti@gmail.com" required class="input">
+                <textarea id="textarea" name="comment" maxlength="500" cols="30" rows="10" placeholder="Write your message here" required class="input text-area"></textarea>
+                <button class="see-more submit" type="submit">Get In Touch</button>
+            </form>
```
Comment (aboongm) [inline index.html:165]: - When clicked on the button, the form details should be sent to formspree website
- The button interaction for hover, active, and disabled should be styled in css


contact-1

Compare your design and the figma design:
1. The background is grey. Please match the figma design
2. The background image seems to be working, but the blue background is the issue. I see you are using an image for the blue background and hence it is not responsive.
3. The margin/padding is not matching the figma design. Please follow the figma design as closely as possible.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    <title>Portfolio-Mobile Version</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--Work section-->
+    <section class="work">
+        <!---The first card -->
+        <div class="flex">
+            <img src="images/card1-image.png" alt="Image for Card 1">
```
Comment (aboongm) [inline index.html]: - Why is the header section removed? Please make the project complete without removing parts of the project.


[OPTIONAL] Your css style of the previous projects are still not according to figma design. Please try to match the figma design in font, font-size, color, margin and padding.

**Review by aboongm — CHANGES_REQUESTED** [review]
Hi @Loltolo-Lesapiti @Dadadon @RiyaBulia12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

request_changes_1

## Highlights
- All linters are passing ✔️
- You are following GitHub actions ✔️
- Good commit history ✔️
- Good PR description ✔️

## Required Changes ♻️

- Please check the comments below for required changes.

## Optional suggestions


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Loltolo-Lesapiti    :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes that were requested by the previous reviewer.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 
## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**